### ISSUE TITLE
In the Feathercoin Core for Mac - bitcoin icon clashes

### ISSUE THREAD

**Issue by wrapperband** [FeatherCoin/Feathercoin#228]
http://forum.feathercoin.com/topic/9152/hello-i-m-new-and-something-strange-with-feathercoin-core-transaction-notification-on-a-mac

Should be possible to make a feathercoin.ico copy (of Bitcoin.ico) and specify that as the system icon in OS system builds.

**Comment by looarn** [comment]
Here what I had to the forum :

I don’t think this is a l10n issue anyway, as this string doesn’t exist :D
Looks to me that the OS is interpreting this field :

**Feathercoin/share/qt/Info.plist.in**

```
  <key>CFBundleExecutable</key>
  <string>Feathercoin-Qt</string>

  <key>CFBundleIdentifier</key>
  <string>org.bitcoinfoundation.Bitcoin-Qt</string>
```

**Comment by tefica** [comment]
hi looarn, my Feathercoin Core version is the 0.9.6.1, my MacBook is in French ;)

**Comment by looarn** [comment]
Look to me that this issue and #204 are linked.

**Comment by aciddude** [comment]
Confirmed fixed in MacOS 0.9.6.2 release build.
